### ISSUE TITLE
WebdriverIO saveScreenshot get the fullpage

### ISSUE THREAD

**Issue by lsq4590578** [Codeception/CodeceptJS#875]
Hi Manito,
    When we written test script,we found saveScreenshot method can't get fullpage,it just get current screen display picture

`I.saveScreenshot('1.png',true)`

**Comment by reubenmiller** [comment]
Looking at the webdriverio api, it looks like it is a bug when setting the windows size

In Codeceptjs file `helper/WebDriverIO.js`, the `resizeWindow` function should be:
```js
  async resizeWindow(width, height) {
    if (width !== 'maximize') {
      return this.browser.windowHandleSize({ width, height });
    }

    const { screenWidth, screenHeight } = await this.browser.execute(() => ({
      height: window.screen.height,
      width: window.screen.width,
    }));

    // return this.browser.windowHandleSize(screenWidth, screenHeight); // This line is wrong!
    return this.browser.windowHandleSize({ width: screenWidth, height: screenHeight });
  }
```
Can you try changing it locally on your side and check if it fixes it? If so then I can do a PR if you want.

**Comment by reubenmiller** [comment]
I have a fix for this now. I’ll see if I can submit a PR soon.

**Comment by reubenmiller** [comment]
Should be fixed in 1.1.4.
